logging/src/windows/logging.cc

```diff
@@ -0,0 +1,52 @@
+#include <leatherman/logging/logging.hpp>
+#include <boost/nowide/iostream.hpp>
+#include <windows.h>
+
+using namespace std;
+
+namespace leatherman { namespace logging {
+    static HANDLE stdHandle;
+    static WORD originalAttributes;
+
+    void colorize(ostream& dst, log_level level)
+    {
+        if (!get_colorization()) {
+            return;
+        }
+
+        // The ostream may have buffered data, and changing the console color will affect any buffered data written
+        // later. Ensure the buffer is flushed before changing the console color.
+        dst.flush();
```

Comment:
I wonder how badly this would hurt performance, but thankfully facter doesn't write out that many lines.